src/app/tests/suites/certification/Test_TC_WNCV_3_2.yaml

```diff
@@ -176,22 +174,14 @@ tests:
               minValue: 10
               maxValue: 42
 
-    ### Depending on the DUT specs we might prefer a arguments here
-    - label: "3a2: DUT updates its attributes"
-      cluster: "DelayCommands"
-      command: "WaitForMs"
-      arguments:
-          values:
-              - name: "ms"
-                value: 2000
```

Comment:
increase to 3000ms if still  failing

Reply:
Sure i will update

Reply:
Even though the delay is increased its still failing